### PR TITLE
chore: add changelog file for 0.4.2 version

### PR DESCRIPTION
**Pull request by marinsalinas**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+## 0.4.2 (Unreleased)
+
+**Fixed bugs:**
+
+- mongodbatlas\_cluster fails to redeploy manually deleted cluster [\#159](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/159)
+
+**Closed issues:**
+
+- mongodbatlas\_alert\_configuration - not able to generate any alerts with event\_type = "OUTSIDE\_METRIC\_THRESHOLD" and matcher.fieldName != "TYPE\_NAME" [\#164](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/164)
+- Still cannot create cluster in region ME\_SOUTH\_1 on plugin version 0.4.1 [\#161](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/161)
```
Comment (themantissa) [inline CHANGELOG.md]: Remove line 10 as same issue as line 12.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+- mongoatlas\_cluster fails to create  - invalid enumeration value M2 was specified  [\#160](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/160)
```
Comment (themantissa) [inline CHANGELOG.md]: remove as was closed by the reporter

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+- Can't create cluster ME\_SOUTH\_1 region [\#157](https://github.com/terraform-providers/terraform-provider-mongodbatlas/issues/157)
+
+**Merged pull requests:**
+
+- Doc: Fix import for mongodbatlas\_project\_ip\_whitelist [\#166](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/166) ([haidaraM](https://github.com/haidaraM))
+- chore: removed wrong validation for matchers.value [\#165](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/165) ([PacoDw](https://github.com/PacoDw))
+- feature: add default label to clusters [\#163](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/163) ([marinsalinas](https://github.com/marinsalinas))
```
Comment (themantissa) [inline CHANGELOG.md]: Please remove feature and just start the line with "Add".

**Review by themantissa — CHANGES_REQUESTED** [review]
A few tweaks to make cleaner and we'll be good.

**Review by PacoDw — APPROVED** [review]
LGTM!

**Review by themantissa — APPROVED** [review]
LGTM with commit with changes.

**Pull request merged**